recipe/install-libarrow.sh

```diff
@@ -81,9 +81,6 @@ elif [[ "${PKG_NAME}" == "libarrow-all" ]]; then
     rm -f $PREFIX/lib/{libarrow_testing,libarrow_flight_testing}.*
     rm -f $PREFIX/lib/pkgconfig/{arrow-testing,arrow-flight-testing}.pc
     rm -rf $PREFIX/lib/cmake/{ArrowTesting,ArrowFlightTesting}
-    # Remove executables
-    rm -f $PREFIX/bin/parquet-*
-    rm -f $PREFIX/bin/arrow-*
```

Comment:
As I said, if you remove this you need to host-depend on the utils in `libarrow-all` as well.